frame/glutton/src/lib.rs

```diff
@@ -105,6 +105,37 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(crate) type TrashDataCount<T: Config> = StorageValue<_, u32, ValueQuery>;
 
+	#[pallet::genesis_config]
+	pub struct GenesisConfig {
+		pub compute: Perbill,
+		pub storage: Perbill,
+		pub trash_data_count: u32,
+	}
+
+	impl Default for GenesisConfig {
+		fn default() -> Self {
+			Self {
+				compute: Default::default(),
+				storage: Default::default(),
+				trash_data_count: Default::default(),
```

Comment:
5000?
https://github.com/paritytech/substrate/blob/polkadot-v0.9.42/frame/glutton/src/lib.rs#L146

Reply:
That's again a statement based on Kusama/Polkadot max PoV. I would keep it as it is, but I can change it to `5000` if you prefer.

Reply:
Default is nicer.